utils/municipality.ts

```diff
@@ -0,0 +1,339 @@
+    denominazione: record[7],
+    denominazioneInItaliano: record[6],
+    denominazioneRegione: record[4],
+    siglaProvincia: ""
+  };
+  return Municipality.decode(municipality);
+};
+
+/**
+ * A string that represents a municipality codice catastale
+ * https://it.wikipedia.org/wiki/Codice_catastale
+ */
+
+const CODICE_CATASTALE_REGEX = `^[A-Z]\\d{3}$`;
+
+export const CodiceCatastale = PatternString(CODICE_CATASTALE_REGEX);
+export type CodiceCatastale = t.TypeOf<typeof CodiceCatastale>;
+
+/**
+ * load all the codici catastali and create a mapping between the name of the municipality and the codice catastale
+ */
+const loadMunicipalityToCatastale = (
+  blobService: BlobService
+): TaskEither<Error, Map<string, string>> =>
+  getFileFromBlob(
+    blobService,
+    MUNICIPALITIES_CONTAINER_NAME,
+    MUNICIPALITIES_CATASTALI_BLOB_NAME
+  )
+    .foldTaskEither<Error, StringMatrix>(
+      err => fromLeft(err),
+      _ =>
+        _.foldL(
+          () => fromLeft(new Error("Municipalities with catastale is empty")),
+          __ =>
+            parseCsv(__, optionMunicipalitiesWithCatastale).foldTaskEither(
+              e => fromEither(leftE(e)),
+              matrixString => fromEither(rightE(matrixString))
+            )
+        )
+    )
+    .foldTaskEither<Error, Map<string, string>>(
+      err => fromLeft(err),
+      municipalitiesCatastaleRows =>
+        fromEither(
+          rightE(
+            municipalitiesCatastaleRows.reduce(
+              (map: Map<string, string>, row) => {
+                map.set(row[1].toLowerCase(), row[0]);
+                return map;
+              },
+              new Map<string, string>()
+            )
+          )
+        )
+    );
+
+const fromAbolishedMunicipalityToSerializableMunicipality = (
+  abolishedMunicipality: t.TypeOf<typeof AbolishedMunicipality>,
+  codiceCatastale: string
+) => {
+  return {
+    codiceCatastale,
+    municipality: {
+      codiceProvincia: "",
+      codiceRegione: "",
+      denominazione: abolishedMunicipality.comune,
+      denominazioneInItaliano: abolishedMunicipality.comune,
+      denominazioneRegione: "",
+      siglaProvincia: abolishedMunicipality.provincia
+    }
+  } as ISerializableMunicipality;
+};
+
+/**
+ * load the abolished municipality and filter the municipality without catastal code
+ * @param blobService: used to to connect to blobStorage
+ * @param municipalityToCatastale: used to filter and remove the municipality without catastal code
+ */
+const loadAbolishedMunicipalities = (
+  blobService: BlobService,
+  municipalityToCatastale: Map<string, string>
+): TaskEither<Error, ReadonlyArray<ISerializableMunicipality>> =>
+  getFileFromBlob(
+    blobService,
+    MUNICIPALITIES_CONTAINER_NAME,
+    ABOLISHED_MUNICIPALITIES_BLOB_NAME
+  )
+    .chain(rawFile =>
+      rawFile.foldL(
+        () => fromEither(leftE(new Error("abolished municipalities is empty"))),
+        json => fromEither(rightE(JSON.parse(json)))
```

Comment:
```suggestion
        json => taskEither.of(JSON.parse(json)))
```

Besides of that, be careful because `JSON.parse` can throw! Consider use [parseJSON](https://github.com/gcanti/fp-ts/blob/1.x/test/Either.ts#L533)